egs/librispeech/asrtts/conf/train_pytorch_tacotron2+spkemb.yaml

```diff
@@ -0,0 +1,47 @@
+# This is the basic tactron2 training settting
+
+# encoder related
+embed-dim: 512
+elayers: 1
+eunits: 512
+econv-layers: 3 # if set 0, no conv layer is used
+econv-chans: 512
+econv-filts: 5
+
+# decoder related
+dlayers: 2
+dunits: 1024
+prenet-layers: 2  # if set 0, no prenet is used
+prenet-units: 256
+postnet-layers: 5 # if set 0, no postnet is used
+postnet-chans: 512
+postnet-filts: 5
+use-speaker-embedding: true
+
+# attention related
+atype: location
+adim: 128
+aconv-chans: 32
+aconv-filts: 15      # resulting in filter-size = aconv-filts * 2 + 1
```

Comment:
We observed a significant performance degradation when we reduce the filter size? Is it OK for your case?

Reply:
Sorry, this is about ASR. Please ignore my comments.

Reply:
ASR should be configured in YAML too.